api/pipelineUsage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Joi = require('joi');
+const pipelineId = require('../models/pipeline').base.extract('id');
+const Scm = require('../core/scm');
+
+module.exports = {
+    get: Joi.array().items(
+        Joi.object({
+            id: pipelineId,
+            name: Joi.string().required(),
+            scmRepo: Scm.repo,
+            lastRun: Joi.alternatives().try(Joi.string().isoDate(), Joi.allow(null)),
+            admins: Joi.object().pattern(Joi.string(), Joi.boolean()).required()
+        })
+    )
```

Comment:
```suggestion
    output: Joi.array().items(
        Joi.object({
            id: pipelineSchema.extract('id').required(),
            name: pipelineSchema.extract('name').required(),
            scmRepo: pipelineSchema.extract('scmRepo').required(),
            lastRun: Joi.alternatives().try(eventSchema.extract('createTime'), Joi.allow(null)),
            admins: pipelineSchema.extract('admins').required()
        })
    )
```

Reply:
I think `get` better matches the naming format we've been using for other files.